src/gerbil/gxi

```diff
@@ -1,11 +1,48 @@
 #!/bin/sh
 set -eu
 
+
 if [ -z "${GERBIL_HOME:-}" ]; then
-    GERBIL_HOME=$(dirname $(cd ${0%/*} && echo $PWD))
+
+    # First, try whereis. We are looking for $GERBIL_HOME/lib/std, our stdlib
+    for gh in $(whereis -b gerbil |cut -d: -f2); do
```

Comment:
how long does this take?

Reply:
Also, on my system where I don't have a system-wide install of gerbil, `whereis` comes empty.
How well will this work in that case?

Reply:
Furthermore, I think it might be the wrong thing to do -- it can break in situation where we have a system-wide install and a user install. `whereis` will find the system-wide install even if we are invoking the user install gxi directly.
I think we should only use the read-link logic (provided it's reliable).

Reply:
readlink (1) is not portable across variants of Unix, (2) will have surprising results in some environments, such as the Google blaze build where every input file is a symlink to the content-addressed store (admittedly a rare case, where hopefully an environment variable can be set).